example/src/main/java/org/nervos/ckb/service/RpcCallback.java

```diff
@@ -0,0 +1,9 @@
+package org.nervos.ckb.service;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+interface RpcCallback<T> {
```

Comment:
I think `RpcCallback` in SDK can be reused.

Reply:
Yes, you are right I will fix this.

Reply:
updated: [`fdb0bd9` (#348)](https://github.com/nervosnetwork/ckb-sdk-java/pull/348/commits/fdb0bd92701d7591e8cd118768f22856de06df45)